twilight-http/src/request/channel/thread/mod.rs

```diff
@@ -12,11 +13,17 @@ mod remove_thread_member;
 mod update_thread;
 
 pub use self::{
-    add_thread_member::AddThreadMember, create_thread::CreateThread,
+    add_thread_member::AddThreadMember,
+    create_forum_thread::{CreateForumThread, ForumThreadMessageParams},
```

Comment:
instead of re-exporting both types, let's just re-export the request type and make the module itself public